packages/v4/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,345 @@
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post2.author} •{" "}
+              {featuredPostsData.post2.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+        <GalleryItem>
+          <Card isFullHeight>
+            <CardHeader
+              style={{
+                minHeight: "200px",
+                backgroundPosition: "center",
+                backgroundSize: "cover",
+                backgroundImage: `url(${featuredPostsData.post3.imageURL})`,
+                height: "300px",
+              }}
+            ></CardHeader>
+            <Divider />
+            <CardTitle>
+              {" "}
+              <a href={featuredPostsData.post3.URL}>
+                {featuredPostsData.post3.title}
+              </a>
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post3.author} •{" "}
+              {featuredPostsData.post3.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+        <GalleryItem>
+          <Card isFullHeight>
+            <CardHeader
+              style={{
+                minHeight: "200px",
+                backgroundPosition: "center",
+                backgroundSize: "cover",
+                backgroundImage: `url(${featuredPostsData.post4.imageURL})`,
+                height: "300px",
+              }}
+            ></CardHeader>
+            <Divider />
+            <CardTitle>
+              {" "}
+              <a href={featuredPostsData.post4.URL}>
+                {featuredPostsData.post4.title}
+              </a>
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post4.author} •{" "}
+              {featuredPostsData.post4.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+      </Gallery>
     </PageSection>
-    <PageSection className="ws-homepage-main-section pf-m-light-100">
-      <Grid>
-        <GridItem sm={12} md={6} mdOffset={3} className="pf-v5-u-py-2xl pf-v5-u-text-align-center" id="about-patternfly-section">
-          <TextContent>
-            <AutoLinkHeader size="h1" headingLevel="h2" className="ws-title">
-              {aboutPatternFly}
-            </AutoLinkHeader>
-            <Text component={TextVariants.p} className="ws-mdx-p">
-              We think quality product experiences should be easy to build and easy to scale, so we provide the tools to make that happen.
-            </Text>
-            <Text component={TextVariants.p} className="ws-mdx-p">
-              From documentation and components to code examples and tutorials, PatternFly has it all—and our community of Flyers help make it better every day.
-            </Text>
-          </TextContent>
-        </GridItem>
-      </Grid>
+    <PageSection className="v5-featured-posts">
+      <Flex flexWrap={{default: "wrap"}}>
```

Comment:
Same as https://github.com/patternfly/patternfly-org/pull/3573/files#diff-2cdc77e412cc8cdb720d52ffc48002065a59bc20573de6b80014ac936fe81abcR200

```suggestion
      <Flex direction={{default: "column", md: "row"}} gap={{default: "gapMd"}}>
```